### PR TITLE
feature -> develop | [FLUTTER] Updated: root directory

### PR DESCRIPTION
**Pull request by we-prajapati-c001**
### [Task#1084 - Documentation For VaahFlutter](https://team.webreinvent.com/backend/collab/project/vaahcms/tasks/209)

Updated root directory

Time invested: hh:mm format | Billable: 27:51 | Non-Billable: 00:00
Billable time is the total time invested till [Task-1084-Update-directory-structure-vaahextendflutter-widgets](https://github.com/webreinvent/vaah-docs/tree/feature/Task-1084-Update-directory-structure-vaahextendflutter-widgets)

Task Video Proof: https://youtu.be/5V4Qjk9t9Uk
Updated: lib directory introductions video: https://youtu.be/R2RoSUs8y7I
Updated: docs for atomic design and controllers: https://youtu.be/N8Y4NuUfl8I

### CODE REVIEW

File: docs/vaahflutter/directory_structure/root/d_lib.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    |   | ...
+    |
+    └───  app_config.dart
+    |
+    └───  main.dart
+```
+
+| **Directory name** | **Documentation link** | **Description** |
+| --- | --- | --- |
+| models | [check this](../lib/models) | This directory is project specific and contains models for the project |
```
Comment (themodernpk) [inline docs/vaahflutter/directory_structure/root/d_lib.md:42]: @we-prajapati-c001 explain that to create models we're using `GetX` link it's documentation

File: docs/vaahflutter/directory_structure/root/d_lib.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| views | [check this](../lib/views) | This directory is also project specific and contains UI screens for the project |
```
Comment (themodernpk) [inline docs/vaahflutter/directory_structure/root/d_lib.md:43]: @we-prajapati-c001 for the view we are using `Atomic Design Methodology`, explain and link it to https://atomicdesign.bradfrost.com/chapter-2/

**Review by themodernpk — CHANGES_REQUESTED** [review]

File: docs/vaahflutter/directory_structure/root/d_lib.md (hunk 1 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/directory_structure/root/d_lib.md:42]: @themodernpk sir, on it. for models we use nothing, for controllers (state management) we are using GetX, so will describe state management and GetX.

File: docs/vaahflutter/directory_structure/root/d_lib.md (hunk 2 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/directory_structure/root/d_lib.md:43]: Okay sir will do.

File: docs/vaahflutter/directory_structure/root/d_lib.md (hunk 1 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/directory_structure/root/d_lib.md:42]: @themodernpk sir, In the next PR we will have the lib section https://youtu.be/Pl238bpgBrI, in that section I think we should update this section.

Comment (we-prajapati-c001) [reply docs/vaahflutter/directory_structure/root/d_lib.md:42]: For this section, if wanted we can add brief info for controllers, state management and GetX

File: docs/vaahflutter/directory_structure/root/d_lib.md (hunk 2 above)
Comment (we-prajapati-c001) [reply docs/vaahflutter/directory_structure/root/d_lib.md:43]: https://youtu.be/N8Y4NuUfl8I

**Pull request merged**